CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -19,6 +23,9 @@ local RunService = game:GetService("RunService")
 local RobloxGui = script.Parent
 local ThirdPartyProductName = nil
 
+local Dialog = require(RobloxGui.Modules.VR.Dialog)
```

Comment:
I don't this we should be requiring VR modules on non VR platforms. They can cascade into other modules and soon we create a bunch of VR junk on someones non-vr PC